frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -0,0 +1,127 @@
+import { RootState } from 'global/state';
+
+import { OpenEdXInstanceDeploymentStatusStatusEnum as DeploymentStatus } from 'ocim-client';
+
+import { Col, Row } from 'react-bootstrap';
+
+import { getDeploymentsAlerts } from 'console/actions';
+import { DeploymentAlertModel } from 'console/models';
+import { WrappedMessage } from 'utils/intl';
+
+import messages from './displayMessages';
+import { ConsolePage } from '../ConsolePage';
+
+import './styles.scss';
+
+interface ActionProps {
+  getAlerts: Function;
+}
+interface Props extends ActionProps {
+  alerts: Array<DeploymentAlertModel>;
+}
+
+interface AlertProps {
+  alert: DeploymentAlertModel;
+}
+
+const iconClassMap = {
+  [DeploymentStatus.Healthy]: 'fa-check-circle',
+  [DeploymentStatus.ChangesPending]: 'fa-check-circle',
+  [DeploymentStatus.Provisioning]: 'fa-info-circle',
+  [DeploymentStatus.Preparing]: 'fa-info-circle',
+  [DeploymentStatus.Unhealthy]: 'fa-times-circle',
+  [DeploymentStatus.Offline]: 'fa-stop-circle'
+};
+
+const statusClassMap = {
+  [DeploymentStatus.Healthy]: 'healthy',
+  [DeploymentStatus.ChangesPending]: 'changes_pending',
+  [DeploymentStatus.Provisioning]: 'provisioning',
+  [DeploymentStatus.Preparing]: 'preparing',
+  [DeploymentStatus.Unhealthy]: 'unhealthy',
+  [DeploymentStatus.Offline]: 'offline'
+};
+
+const Alert: React.FC<AlertProps> = (props: AlertProps) => {
+  const [detailsCollapsed, setDetailsCollapsed] = React.useState(true);
+
+  const deploymentStatus = props.alert.status;
+
+  const caretClass = detailsCollapsed ? 'fa-caret-down' : 'fa-caret-up';
+  const collapsedClass = detailsCollapsed ? 'collapsed' : '';
+  const iconClass = iconClassMap[deploymentStatus];
+  const statusClass = statusClassMap[deploymentStatus];
+
+  const month = props.alert.date.toLocaleString('default', { month: 'long' });
+  const day = props.alert.date.getDate();
+  const year = props.alert.date.getFullYear();
+
+  const details = props.alert.deployedChanges ? (
+    JSON.stringify(props.alert.deployedChanges)
+  ) : (
+    <WrappedMessage id="NoDetails" messages={messages} />
+  );
+
+  return (
+    <div className="deployment-alert mb-3">
+      <div className={`p-3 d-flex deployment-alert-header ${statusClass}`}>
+        <div className="d-flex">
+          <i className={`mr-3 align-self-center fa ${iconClass}`} />
+        </div>
+        <div>
+          <p className="m-0 deployment-alert-message">
+            <WrappedMessage id={deploymentStatus} messages={messages} />
+          </p>
+          <p className="m-0 deployment-alert-date">
+            <small>{`${month} ${day}, ${year}`}</small>
+          </p>
+        </div>
+        <div // eslint-disable-line jsx-a11y/interactive-supports-focus
+          role="button"
+          className="d-flex expand-caret-button ml-auto"
+          onClick={() => setDetailsCollapsed(!detailsCollapsed)}
+          onKeyDown={() => setDetailsCollapsed(!detailsCollapsed)}
+        >
```

Comment:
Could we reuse [this](https://github.com/open-craft/opencraft/blob/1ea068526157dfcce6960115f42c6de9ff2f7db5/frontend/src/ui/components/CollapseEditArea/CollapseEditArea.tsx#L20-L27) or make something similar (using proper links instead of `div` button)?